library/neo4j

```diff
@@ -0,0 +1,7 @@
+# maintainer: Ben Butler-Cole <ben@neotechnology.com> (@benbc)
+
+2.3.0-M03: git://github.com/neo4j/docker-neo4j@660ace32a9851511874ce56731446b1424bfc5cf 2.3.0-M03
+latest: git://github.com/neo4j/docker-neo4j@660ace32a9851511874ce56731446b1424bfc5cf 2.3.0-M03
+
+2.3.0-M03_enterprise: git://github.com/neo4j/docker-neo4j@660ace32a9851511874ce56731446b1424bfc5cf 2.3.0-M03_enterprise
+latest_enterprise: git://github.com/neo4j/docker-neo4j@660ace32a9851511874ce56731446b1424bfc5cf 2.3.0-M03_enterprise
```

Comment:
Mind swapping these tags to use `-` instead of `_` and changing `latest_enterprise` to just `enterprise`?